src/forklift/models.py

```diff
@@ -320,7 +320,8 @@ def __init__(
         #: the full path to the source data
         self.source = join(source_workspace, source_name)
 
-        if '.sde' in self.source.lower():
+        #: try to find the full-qualified source name if it doesn't have any "."s in it
+        if '.sde' in self.source.lower() and '.' not in self.source_name:
```

Comment:
**question**: are we implying that if there is a period in the name, then the name is qualified?

Reply:
Yes.